docs/setup.md

```diff
@@ -121,7 +121,7 @@ The Settings Page is available from the **File** menu under **Settings** (Click
    - **Hide Menu Bar** (Windows, Linux)
       - This option hides the menu bar. Press "Alt" to show it.
    - **Show Icon on Menu Bar** (OS X)
```

Comment:
```suggestion
   - **Show Icon on Menu Bar** (macOS)
```

Reply:
can't apply from GitHub itself, the changed line is "Outdated"